Clockify/PluginSettings.cs

```diff
@@ -17,8 +17,11 @@ public class PluginSettings
         public string TaskName { get; set; } = string.Empty;
 
         [JsonProperty(PropertyName = "timerName")]
-        public string TimeName { get; set; } = string.Empty;
+        public string TimerName { get; set; } = string.Empty;
         
+        [JsonProperty(PropertyName = "titleFormat")]
+        public string TitleFormat { get; set; } = "{projectName}:\n{taskName}\n{timer}";
```

Comment:
In order to be compatible to previous version, albeit less discoverable, I'd set the default for `TitleFormat` to `string.Empty`

See also the comment in ToggleAction.cs

Reply:
If this is empty, then it should be empty in PluginActionPI.html too, right?